tox.ini

```diff
@@ -212,8 +213,14 @@ commands=
       ceph_dev_sha1={env:CEPH_DEV_SHA1:latest} \
   "
 
+  # Wait 5 minutes for services to be ready
+  sleep 300
+  testinfra -n 4 --sudo -v --connection=ansible --ansible-inventory={changedir}/hosts {toxinidir}/tests/functional/tests
```

Comment:
I don't believe ``testinfra`` can be run before ``setup.yml`` because it installs packages that testinfra needs to test with.